Makefile

```diff
@@ -258,29 +258,37 @@ endif
 GPRINST_OPTS=-p -f --prefix=$(TPREFIX) \
 	--build-var=LIBRARY_TYPE --build-var=AWS_BUILD
 
-install-native: install-clean
+install-lib-native:
 	$(GPRINSTALL) $(GPROPTS) $(GPRINST_OPTS) $(GPR_DEFAULT) \
 		--build-name=$(DEFAULT_LIBRARY_TYPE) aws.gpr
-	$(GPRINSTALL) $(GPROPTS) $(GPRINST_OPTS) $(GPR_STATIC) --mode=usage \
-		--build-name=$(DEFAULT_LIBRARY_TYPE) \
-		--install-name=aws tools/tools.gpr
 ifeq (${ENABLE_SHARED}, true)
 	$(GPRINSTALL) $(GPROPTS) $(GPRINST_OPTS) \
 		$(GPR_OTHER) --build-name=$(OTHER_LIBRARY_TYPE) aws.gpr
 endif
 
-install-cross: install-clean
+install-tools-native:
+	$(GPRINSTALL) $(GPROPTS) $(GPRINST_OPTS) $(GPR_STATIC) --mode=usage \
+		--build-name=$(DEFAULT_LIBRARY_TYPE) \
+		--install-name=aws tools/tools.gpr
+
+install-native: install-lib-native install-tools-native
```

Comment:
Did you mean to add an `install-clean` here?

Reply:
Ah yes, sorry… Should be fixed now, thanks!